csci_tool/commands/mutate.py

```diff
@@ -72,9 +73,9 @@ def clone(student):
         # commit and push our changes
         for student, repo_path in repos:
             l.info('Committing repo: %s', student.unix_name)
-            repo = GitRepo(repo_path)
+            repo = repo_path
```

Comment:
This should be renamed to repo not repo_path, because its not really a path its a repo